Source/StrokeFormatMenu/ColorCollectionViewCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ColorCollectionViewCell.swift
+//  FolioReaderKit
+//
+//  Created by Catalina Sanchez on 3/21/19.
+//  Copyright © 2019 FolioReader. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ColorCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: - Properties -
+    
+    @IBOutlet weak var colorEllipse: UIImageView!
+    
+    var colorImage: UIColor? {
+        didSet {
+            guard colorImage != nil else { return }
```

Comment:
hacer mejor guard let aca y evitar el force unwrap
```suggestion
            guard let colorImage = colorImage else { return }
```

Reply:
Listo!